clientv3/client.go

```diff
@@ -26,8 +28,13 @@ import (
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/grpclog"
 )
 
+func init() {
```

Comment:
user should be able to set logger themselves. (we by default disable gRPC logging)
probably add the logger interface into Config.